src/main/java/com/opentok/OpenTok.java

```diff
@@ -61,15 +61,15 @@
      * @param apiSecret Your OpenTok API secret. (See the <a href="https://dashboard.tokbox.com">OpenTok dashboard</a>
      * page.)
      */
-    public OpenTok(int apiKey, String apiSecret) {
+    private OpenTok(int apiKey, String apiSecret) {
```

Comment:
I think we can keep one public constructor with just the mandatory parameters. This will be the most concise way to initialize the object and will work for most likely 80% of users. The Builder will just be for those who want to customize by using the additional option parameters.